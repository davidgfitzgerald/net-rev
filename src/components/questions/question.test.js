import React from 'react';
import ReactDOM from 'react-dom'
import Enzyme, { shallow, mount } from 'enzyme';
import toJson from "enzyme-to-json";
import Adapter from 'enzyme-adapter-react-16';
import { unmountComponentAtNode, render, cleanup } from 'react-dom'
import { act } from "@testing-library/react";

Enzyme.configure({adapter: new Adapter()})

import Question from "./question";
import QuestionContainer from "./question_container";

let container = null;
beforeEach(() => {
  container = document.createElement("div")
  document.body.appendChild(container)
})

afterEach(() => {
  unmountComponentAtNode(container)
  container.remove();
  container = null;
})

test.only("QuestionContainer renders (enzyme)",() => {
  const qContainer = render(<QuestionContainer/>, container)
  const wrapper = mount(qContainer)
  expect(toJson(wrapper)).toMatchSnapshot()
  // expect(qContainer).toBeTruthy()
  // console.dir(qContainer)
})

