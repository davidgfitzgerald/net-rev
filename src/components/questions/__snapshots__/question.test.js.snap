// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionContainer renders (enzyme) 1`] = `
<div
  class="question-container"
>
  <h3>
    Question
  </h3>
  <p>
    Loading..
  </p>
</div>
`;
